compiler/rustc_middle/src/query/on_disk_cache.rs

```diff
@@ -492,9 +492,7 @@ impl<'a, 'tcx> CacheDecoder<'a, 'tcx> {
                 // expansion, so we use `import_source_files` to ensure that the foreign
                 // source files are actually imported before we call `source_file_by_stable_id`.
                 if source_file_cnum != LOCAL_CRATE {
-                    self.tcx
-                        .cstore_untracked()
-                        .import_source_files(self.tcx.sess, source_file_cnum);
+                    let () = self.tcx.import_source_files(source_file_cnum);
```

Comment:
```suggestion
                    self.tcx.import_source_files(source_file_cnum);
```
Not sure why the `()`.

Reply:
🙈 because of the `must_use` that is set on all hooks